server/docker.js

```diff
@@ -2,8 +2,15 @@ const axios = require("axios");
 const { R } = require("redbean-node");
 const version = require("../package.json").version;
 const https = require("https");
+const fs = require("fs");
 
 class DockerHost {
+
+    static CertificateBasePath = process.env.DOCKER_TLS_DIR_PATH || "data/docker-tls/";
```

Comment:
Thanks for the pr.

Since `data/` can be changed by the env var `DATA_DIR`, `data/` cannot be hardcoded here.

But actually, in my opinion, these 4 env vars are not necessary, just hardcode:
 - docker-tls
 - ca.pem
 - cert.pem
 - key.pem

Other sub-directories are created in this function.
https://github.com/louislam/uptime-kuma/blob/cdb38d49ebaedbe28ab160ff17e9385826df9893/server/database.js#L93-L108